package.json

```diff
@@ -42,12 +43,12 @@
     "pug": "2.0.3",
     "pug-loader": "2.4.0",
     "rimraf": "2.6.3",
-    "semistandard": "13.0.1",
+    "semistandard": "^13.0.1",
```

Comment:
We should revert his part - otherwise unit tests might break randomly

Reply:
Oh, that snuck in. Will fix.